internal/api/core/routes.go

```diff
@@ -17,3 +18,11 @@ func MakeFullAPIPath(suffix string) string {
 
 	return CURRENT_PREFIX + "/" + suffix
 }
+
+func makeAbsLocalAPIPath(suffix string) string {
```

Comment:
I already mentioned in an earlier comment moving this to another file.
But you should also note that it seems strange that there is only one file that uses this and the function is not located in that file.